### PR TITLE
Changes for 1.7

### PR DESCRIPTION
**Pull request by jgautam**

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/292#" title="Author self-approved">jgautam</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/docs/content/04-clusters/1-new-clusters.md
Comment (saamalik) [inline packages/docs/content/04-clusters/1-new-clusters.md:20]: Should day-2 be Day-2 ? not sure.

**Review by saamalik — APPROVED** [review]

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/292#" title="Author self-approved">jgautam</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/292#pullrequestreview-579843769" title="Approved">saamalik</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: packages/docs/content/04-clusters/1-new-clusters.md
Comment (jgautam) [reply packages/docs/content/04-clusters/1-new-clusters.md]: Yes. Made the change

**Pull request merged**
